src/Makefile.am

```diff
@@ -82,7 +82,8 @@ AM_CPPFLAGS += \
 forked_daapd_LDADD = \
 	$(FORKED_LIBS) \
 	$(FORKED_OPTS_LIBS) \
-	$(COMMON_LIBS)
+	$(COMMON_LIBS) \
+	-lm
```

Comment:
Yes, I'll remove this after "math.h" is removed. Sorry it's my first attempt to work with "configure.ac" was a lack of experience. I looked in configure.ac but wasn't certain where to add it.